### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by salomondush**
- initial setup
- user registration
IMG_2566
- separating library, network, and database email
- Some commands for the library functionality
      - List books
      - Donate books

### CODE REVIEW

File: changeppbot.py [hunk 1]
```diff
@@ -53,20 +56,21 @@ def catch_basic_responses(msg, email, db):
...
     return {
         "is_admin": is_admin,
-        "id": db.get_mavenlink_id(email)
     }
 
 
 def process(client: SocketModeClient, req: SocketModeRequest):
     """This function handles different events from the Slack API and is the starting point for everything that IncludeBot does"""
 
+    print("\n\n\nReceived request: " + str(req))
```
Comment (lunstb) [inline changeppbot.py]: this isn't how we should handle logging, see line 107 of this file

File: commands.md [hunk 2]
```diff
@@ -61,12 +61,19 @@ SlackBot:
 
 4. `internship add_resume`
 
-5. `internship add_company year`
+5. `inernship add_company year`
```
Comment (lunstb) [inline commands.md]: typo

File: commands.md [hunk 3]
```diff
@@ -61,12 +61,19 @@ SlackBot:
...
+View resumes (list)
+
+View resume (individual)
+
+Upload resume
 
-## Networking
+## Networking Module
+
+**New Idea:** Referrals
```
Comment (lunstb) [inline commands.md:76]: YES!

File: lib/modules/databasemodule/databasemodule.py [hunk 4]
```diff
@@ -63,59 +61,73 @@ def interpret_message(self, msg, email, db, admin, mavenlink_id) -> Dict:
                 return {
                     "command": Commands.ADMIN_LIST_USERS
                 }
+        elif commands[0] == "register": ## TODO: I saw no reason to create new module since this is user related
```
Comment (lunstb) [inline lib/modules/databasemodule/databasemodule.py]: I think I agree with this for now but leave TODO: potentially move user management to new module. I think it's good for rn but if it scales in functionality we will want to split it

File: lib/modules/modulehelpers.py [hunk 5]
```diff
@@ -11,9 +12,22 @@ def get_dates(msg):
     dates = []
     for match in matches:
         dates.append(match)
-
     return dates
 
+def get_book_name_from_isbn(isbn: str):
```
Comment (lunstb) [inline lib/modules/modulehelpers.py:17]: This is good, it would be great if there was a link to good docs explaining the api, just add a line here :)

**Review by lunstb — CHANGES_REQUESTED** [review]

File: commands.md (hunk 2 above)
Comment (salomondush) [reply commands.md]: fixed

File: changeppbot.py (hunk 1 above)
Comment (salomondush) [reply changeppbot.py]: fixed

File: lib/modules/modulehelpers.py (hunk 5 above)
Comment (salomondush) [reply lib/modules/modulehelpers.py:17]: Added

File: lib/modules/databasemodule/databasemodule.py (hunk 4 above)
Comment (salomondush) [reply lib/modules/databasemodule/databasemodule.py]: left a todo as requested

**Review by lunstb — APPROVED** [review]

**Pull request merged**